app/views/property_details.html

```diff
@@ -0,0 +1,192 @@
+{% extends "layout.html" %}
+{% from "govuk/components/phase-banner/macro.njk"  import govukPhaseBanner %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/footer/macro.njk" import govukFooter %}
+
+{% block pageTitle %}
+Apply for legal aid -
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: "prototype"
+    },
+    html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+  }) }}
+
+  {{ govukBackLink({
+    text: "Back",
+      href: "property"
+  }) }}
+{% endblock %}
+
+{% block content %}
+
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form action="vehicles" method="post" class="form">
+      <div class="govuk-form-group">
+        <fieldset class="govuk-fieldset">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+            <h1 class="govuk-fieldset__heading">
+            Your client's property</h1>
+            </legend>
+<br>
+
+              <div class="govuk-form-group">
+                <h1 class="govuk-label-wrapper">
+                  <label class="govuk-label govuk-label--m" for="cost">
+                    How much is your client's home worth?
+                    <div>
+                    <span id="postcode-hint" class="govuk-hint">
+                      You can use property websites to find the estimated value.
+                    </span>
+                    </div>
+                  </label>
+                </h1>
+                <div class="govuk-input__wrapper">
+                  <div class="govuk-input__prefix" aria-hidden="true">£
+                  </div>
+                  <input class="govuk-input govuk-input--width-5" id="cost" name="cost" type="text" spellcheck="false">
+                </div>
+              </div>
+
+              <div class="govuk-form-group">
+                <h1 class="govuk-label-wrapper">
+                  <label class="govuk-label govuk-label--m" for="cost">
+                    What is the outstanding mortgage on your client's home?
+                    <div>
+                    <span id="postcode-hint" class="govuk-hint">
+                      You can use property websites to find the estimated value.
+                    </span>
+                    </div>
+                  </label>
+                </h1>
+                <div class="govuk-input__wrapper">
+                  <div class="govuk-input__prefix" aria-hidden="true">£
+                  </div>
+                  <input class="govuk-input govuk-input--width-5" id="cost" name="cost" type="text" spellcheck="false">
+                </div>
+              </div>
+
```

Comment:
This section (outstanding mortgage) looks like a duplicate?

Reply:
You are right - I have fixed this now! Thank you